classification2.Rmd

```diff
@@ -186,15 +214,15 @@ knn_fit <- workflow() |>
 knn_fit
 ```
 
-> Note: Here again you see the `set.seed` function. In the K-nearest neighbours algorithm, 
-> there is a tie for the majority neighbour class, the winner is randomly selected. Although there is no chance
+> Note: Here again you see the `set.seed` function. In the $K$-nearest neighbors algorithm, 
+> if there is a tie for the majority neighbor class, the winner is randomly selected. Although there is no chance
 > of a tie when $K$ is odd (here $K=3$), it is possible that the code may be changed in the future to have an even value of $K$. 
 > Thus, to prevent potential issues with reproducibility, we have set the seed. Note that in your own code,
```

Comment:
this note block is quite odd. I'm not sure what it's saying

also "you only have to" -> "you should only" -- we should be very clear that in regular data analyses you set.seed once at the beginning, but for our textbook we've used it more to help us carefully create examples for learning purposes